lib/install.mjs

```diff
@@ -0,0 +1,132 @@
+import { createWriteStream, existsSync, rmSync, renameSync } from 'node:fs';
+import { get } from 'node:https';
+import { dirname, resolve } from 'node:path';
+import { arch, env, platform, exit } from 'node:process';
+import { fileURLToPath, URL } from 'node:url';
+
+import compressing from 'compressing';
+import progress from 'cli-progress';
+import semver from 'semver';
+
+import nodeManifest from "../package.json" assert { type: "json" };
+
+/**
+ * NW.js build flavor
+ * 
+ * @type {'sdk' | 'normal'}
+ */
+let buildType = env.npm_config_nwjs_build_type || env.NWJS_BUILD_TYPE || 'normal';
+
+// Parse Node manifest version.
+let parsedVersion = semver.parse(nodeManifest.version);
+let versionString = `${parsedVersion.major}.${parsedVersion.minor}.${parsedVersion.patch}`;
```

Comment:
**IGNORABLE:** A slightly different way to write this that emphasizes the symmetry of the values, removes the noisy characters (\`${}${}${}\`), and ensures the repetitive string that joins the values is consistent (via join).
```suggestion
let versionString = [
  parsedVersion.major,
  parsedVersion.minor,
  parsedVersion.patch
].join('.');
```